dev/io.openliberty.microprofile.telemetry.1.0.internal.jaeger_fat/fat/src/io/openliberty/microprofile/telemetry/internal/tests/AgentConfigTest.java

```diff
@@ -96,10 +103,9 @@ public static void setup() throws Exception {
     @Before
     public void resetServer() throws Exception {
         // Replace any test files with their original versions
-        copyToServer("server.xml-original", "server.xml");
-        copyToServer("jvm.options-original", "jvm.options");
-        copyToServer("bootstrap.properties-original", "bootstrap.properties");
         deleteFromServer("agent-config.properties");
+        deleteFromServer("jvm.options");
+        deleteFromServer("bootstrap.properties");
```

Comment:
I can see why you changed the `server.xml` so that we don't copy and edit it for each test, that makes doing repeats much easier, but I'm not sure why you changed the way `bootstrap.properties` and `jvm.options` are configured.